global-api/routes/city_context.py

```diff
@@ -0,0 +1,72 @@
+from fastapi import APIRouter, HTTPException
+from sqlalchemy import text
+from db.database import SessionLocal
+
+api_router = APIRouter(prefix="/api/v0")
+
+def db_city_context(locode: str):
+    with SessionLocal() as session:
+        query = text(
+            """
+            SELECT a.locode, cp.city_name, cp.region_code,
+            a.region_name as region_name,
+            MAX(CASE WHEN attribute_type = 'population' THEN attribute_value END)::numeric AS population,
+            MAX(CASE WHEN attribute_type = 'population density' THEN attribute_value END)::numeric AS population_density,
+            MAX(ROUND(ST_Area(ST_Transform(geometry, 3857)) / 1000000)) AS area_km2,
+            MAX(CASE WHEN attribute_type = 'elevation' THEN attribute_value END)::numeric AS elevation,
+            MAX(CASE WHEN attribute_type = 'main biome' THEN attribute_value END) AS biome,
+            MAX(CASE WHEN attribute_type = 'income' THEN attribute_value END) AS low_income,
+            MAX(CASE WHEN attribute_type = 'inadequate water access' THEN attribute_value END) AS inadequate_water_access,
+            MAX(CASE WHEN attribute_type = 'inadequate sanitation' THEN attribute_value END) AS inadequate_sanitation
+            FROM modelled.city_attribute a
+            INNER JOIN modelled.city_polygon cp
+            ON a.locode = cp.locode
+            WHERE REPLACE(a.locode, ' ', '') = REPLACE(:locode, ' ', '')
```

Comment:
Would be nice to standardize this in the API so the version with the space is used everywhere, but works for now since that's the data you're getting in the API call.